src/stockwatch/use_cases/importing.py

```diff
@@ -72,13 +72,29 @@ def _process_sell_transaction_row(
     isin: IsinStr,
     row: dict[str, str],
     exchanges: list[CurrencyExchange],
+    cash_settlements: list[CashSettlement],
 ) -> ShareTransaction:
     descr = row["Omschrijving"].split()
     key_index = descr.index("Verkoop")
     nr_stocks = float(descr[key_index + 1].replace(",", "."))
     curr = row["Mutatie"]
     price = Amount(float(descr[key_index + 3].replace(",", ".")), curr)
     amount = apply_exchange(transaction_datetime, nr_stocks * price, exchanges)
+    # process delistings:
+    # so far only a single example of delisting has been presented
+    # for this one, it holds that the amount related to the sell transaction representing
+    # the delisting is found in a cash_settlement (and that one was in USD)
+    if amount.value == 0.0 and "DELISTING:" in descr:
+        if the_settlement := next(
```

Comment:
I have to say i find:
```
for elem in cash_settlements:
    if elem.isin == isin and elem.settlement_date == transaction_datetime.date():
        amount = the_settlement.amount
        break
else:
    print("Some error log")
```
a bit more intuitive then with the `next` usage here.

Reply:
Done